definitions/infra-azurecosmosdbcollection/summary_metrics.yml

```diff
@@ -3,17 +3,24 @@ providerAccountName:
     key: providerAccountName
   title: Azure Account
   unit: STRING
-totalRequests:
+requests:
```

Comment:
based on the other files. are the metrics here missing the `facet: entityName` ?

Reply:
If I'm not wrong, `facet: entityName` has to be specified on golden metrics files, right?

Reply:
totally true. my bad!